### PR TITLE
Add pt-BR translation

### PR DESCRIPTION
**Pull request by jgpacker**

### CODE REVIEW

File: extension/dist/i18n/pt-BR.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "Advanced options": "Opções avançadas",
+  "Copied": "copiado!",
```
Comment (viniciuskneves) [inline extension/dist/i18n/pt-BR.json]: Shouldn't ``copiado!`` be ``Copiado!``?

Comment (jgpacker) [reply extension/dist/i18n/pt-BR.json]: it's lowercase because it is lowercase in the english and spanish translations

**Comment by edouard-lopez** [comment]
Thanks @jgpacker for the work! 

Could you submit your PR on the [lesspass/pure](https://github.com/lesspass/pure) so it will be integrated everywhere (browser, website, android, etc.)
Note there is a thread for translation on the main repo: lesspass/lesspass#150

**Comment by jgpacker** [comment]
Alright! Added https://github.com/lesspass/pure/pull/20

**Pull request closed without merging**